LocalSettings.php

```diff
@@ -4269,6 +4269,10 @@
 		'wmgUseUploadWizard' => [
 			'campaignExpensiveStatsEnabled' => false,
 		],
+		'+gpcommonswiki' => [
```

Comment:
I'd so much be inclined to setting this UploadWizard on a per-wiki basis on LocalWiki.php though

Reply:
This should work how you have it here.

Reply:
> This should work how you have it here.

Ah, okay, I thought it wasn't going to though.

Reply:
It is though we should probably add this to ManageWiki, or have someone at Miraheze generate a simple Flickr key to use globally.

Reply:
> It is though we should probably add this to ManageWiki, or have someone at Miraheze generate a simple Flickr key to use globally.

Yeah, it would be necessary at this point to have the configurations added to ManageWiki, as for the Flickr key, a global one is indeed a good idea, but anyone could use theirs if they wish.